assets/csidriveroperators/aws-ebs/guest_assets/10_cr.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operator.openshift.io/v1
```

Comment:
nits:

* remove `assets` from `guest_assets` / `mgmt_asset` - they're in `assets` dir already.
* directory names should have `hypershift` somewhere, `assets/csidriveroperators/aws-ebs/hypershift/guest`?